src/savegame.cpp

```diff
@@ -172,6 +172,11 @@ void game::unserialize( std::istream &fin )
         data.read( "calendar_start", tmpcalstart );
         calendar::initial_season = static_cast<season_type>( data.get_int( "initial_season",
                                    static_cast<int>( SPRING ) ) );
+        // 0.E stable
+        if( savegame_loading_version < 27 ) {
```

Comment:
Then I think it should be `< 26` here? Otherwise any saves before this PR gets merged (version <= 26) still gets their time multiplied by 6.

Reply:
no it means i'll need to make another savegame bump pr. this is still a draft because i need to be sure that i'm hitting all the bases